### DISCUSSION TITLE
Welcome to yolov5-opencv-cpp-python Discussions!

### DISCUSSION THREAD

**Discussion by doleron** [doleron/yolov5-opencv-cpp-python#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GreenAvocado92** [comment]
Hi, I use it to deploy yolov5s onnx model， I can run the cpp code, but the results is wrong, I do not know how to solve it

**Reply by BilalAzhar0** [reply]
Are you using the trained neural network files in the config directory or did you train your own neural network?

**Comment by BilalAzhar0** [comment]
Hi, in the yolo-tiny.py example, line 36:
`for r in range(25200):`
Why is it iterated 25,200 times? How do you know that the output_data[] has that many elements? Is there something I do not understand regarding what the output "predictions[0]" looks like? Can you perhaps refer me to some sort of documentation? Thank you!

**Comment by 13520141436** [comment]
Can this yolov5-opencv-cpp-python project run successfully on ARM based platform? I want to run it on ARM64 V8.2 + Nvidia A100(or P4000). Thanks a lot!
